pages/index.js

```diff
@@ -2,33 +2,35 @@
 import Head from 'next/head'
 import Link from 'next/link'
 
-import {AppBar, Box, Toolbar, Typography, Button} from '@mui/material';
+import { AppBar, Box, Toolbar, Typography, Button } from '@mui/material';
 
 import styles from '../utils/styles/NavLayoutStyles/HomePage.module.css';
 
 export default function LandingPage() {
   return (
     <div className={styles.homePage}>
-    <Box sx={{ flexGrow: 1 }}>
-    <AppBar className={styles.homeAppBar} position="static">
-      <Toolbar>
-        <Typography className={styles.siteTitle} variant="h4" component="div" sx={{ flexGrow: 1 }}>
-          ClassMates
-        </Typography>
-        <Link href="/login">
-          <Button color="inherit" className={styles.loginButton}>Login</Button>
-        </Link>
-        <Link href="/signup">
-          <Button color="inherit" className={styles.signupButton}>Sign Up</Button>
-        </Link>
-      </Toolbar>
-    </AppBar>
-  </Box>
+      <Box sx={{ flexGrow: 1 }}>
+        <AppBar className={styles.homeAppBar} position="static">
+          <Toolbar>
+            <Typography className={styles.siteTitle} variant="h4" component="div" sx={{ flexGrow: 1 }}>
+              ClassMates
+            </Typography>
+            <Link href="/login">
```

Comment:
Double check documentation but I think you need to add in the prop `passHref` if the children of Link is a custom component

https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag